README.md

```diff
@@ -176,8 +212,8 @@ You can also disable strict mode globally using a [global configuration](#global
 To add a custom parser, first create a parser class:
 
 ```rb
-# app/parsers/truncate_string_parser.rb
-class TruncateStringParser < Decanter::Parser::ValueParser
+# app/parsers/truncated_string_parser.rb
+class TruncatedStringParser < Decanter::Parser::ValueParser
```

Comment:
Earlier in the ReadME (in the parser generator documentation), this was referred to as `TruncatedStringParser`, so I updated this name here to match. It seems like the rest of the parsers are named as nouns rather than verbs. I can revert these changes if they aren't needed.

Reply:
Nice catch!